simulate/simulateIntergenic.go

```diff
@@ -204,3 +214,47 @@ func SimulateWithIndels(fastaFile string, branchLength float64, propIndel float6
 
 	return answer
 }
+
+// changeBaseTransitionBias substitutes an input base b following the K80 model with a transitionBias parameter gamma.
+func changeBaseTransitionBias(b dna.Base, gamma float64) dna.Base {
+	var rand = rand.Float64()
+	var TvProb float64 = 1.0 / (2.0 + gamma) //p(a -> c) = 1 / (2 + gamma) (transversion)
+	switch dna.ToUpper(b) {
+	case dna.A:
```

Comment:
Why does dna.A never return dna.A, and can only return C or T or G? Does this mean there will definitely be a substitution (as opposed to no substitution, A still returns A)?